src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -33,6 +35,12 @@
 
     RequestSpecification bearerTokenForExternalRole;
     RequestSpecification bearerTokenForNonPuiUserManager;
+    String orgId;
+
+    @Before
+    public void setUp() {
+        orgId = createAndUpdateOrganisationToActive(hmctsAdmin);

```

Comment:
@Before will be called each time before test case, so optimisation will not happen. So orgId can be made static and checked every time if its null then only call method createAndUpdateOrganisationToActive()

Reply:
do it for all classes

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/835/commits/d5fc8f7a1d2aef91c119a242e3e00fa40dda4627